### ISSUE TITLE
SIGABRT when pressing on the log in button

### ISSUE THREAD

**Issue by tnguyen42** [facebook/react-native-fbsdk#18]
Hi,

I've encountered a new issue: when pressing the login button, the app freezes and XCode returns a SIGABRT error in main.m, highlighting the return line of it:

```
int main(int argc, char * argv[]) {
  @autoreleasepool {
    return UIApplicationMain(argc, argv, nil, NSStringFromClass([AppDelegate class]));
  }
}
```

For info, I successfully logged out just before.
Does anyone know why that? And what could help?

**Comment by philikon** [comment]
Can you please provide a full printout of the stack trace and/or screenshot? Thanks!

**Comment by tnguyen42** [comment]
Ok, indeed the stack trace (I believe this is the stack trace at least) indicates me that:
"2015-08-25 23:44:08.480 JibberJabber[41708:1868552] *** Terminating app due to uncaught exception 'InvalidOperationException', reason: 'App ID not found. Add a string value with your app ID for the key FacebookAppID to the Info.plist or call [FBSDKSettings setAppID:].'"

The error came from the fact that I tried several positions to insert the code in plist.info (I never did XML), and I did put it between the <dict> and </dict> brackets lower in the code and not in the first one.
Thanks for your help!
